apps/web/playwright/booking-limits.e2e.ts

```diff
@@ -40,7 +40,7 @@ const BOOKING_LIMITS_MULTIPLE = {
 // prevent tests from crossing year boundaries - if currently in Oct or later, start booking in Jan instead of Nov
 // (we increment months twice when checking multiple limits)
 const firstDayInBookingMonth =
-  dayjs().month() >= 9 ? dayjs().add(1, "year").month(0).date(1) : dayjs().add(1, "month").date(1);
+  dayjs().month() >= 8 ? dayjs().add(2, "year").month(0).date(1) : dayjs().add(2, "month").date(1);
```

Comment:
For some reason this was always returning 2023 for me 1 when I changed to +2 it was returning the correct date time we expected